templates/compliance/components/RulesTable.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {
+  TableComposable,
+  Thead,
+  Tbody,
+  Th,
+  Tr,
+  Td,
+} from '@patternfly/react-table';
+
+export interface RulesInterface {
+  topRules: {
+    compliant: boolean,
+    identifier: string,
+    refId: string,
+    remediationAvailable: boolean,
+    severity: string,
+    systemsCount: number,
+    title: string,
+    _typename: string,
+  }[]
+}
+
+const RulesTable: React.FC<RulesInterface & React.HTMLAttributes<HTMLDivElement>> = ({ topRules }) => {
+  return (
+    <TableComposable variant='compact'>
+      <Thead>
+        <Tr>
+          <Th>Rule name</Th>
+          <Th>ID</Th>
+          <Th>Severity</Th>
+          <Th>Failed systems</Th>
+        </Tr>
+      </Thead>
+      <Tbody>
+        {topRules.map((ruleObj, index) => (
+          <Tr key={index} {...(index % 2 === 0 && { isStriped: true })}>
```

Comment:
AFAIK you should be able to pass the `isStriped` prop to the `TableComposable` component and will do this for you.